MapboxSearch/sdk/api/sdk.api

```diff
@@ -15,8 +15,6 @@ public final class com/mapbox/search/BuildConfig {
 	public static final field BUILD_TYPE Ljava/lang/String;
 	public static final field DEBUG Z
 	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
-	public static final field VERSION_CODE I
-	public static final field VERSION_NAME Ljava/lang/String;
```

Comment:
`VERSION_NAME` and `VERSION_CODE` have been removed from BuildConfig after gradle update https://developer.android.com/studio/releases/gradle-plugin?hl=hu#version_properties_removed_from_buildconfig_class_in_library_projects

I've reverted `VERSION_NAME` only to `:sdk-common` because it's our internal module and is not accessible to customers (unless they add it explicitly to their projects). I'm not sure whether these values should be in our public API but in general for SDKs everything should be private or internal unless proved otherwise.